apigw-rest-api-lambda-node/README.md

```diff
@@ -0,0 +1,119 @@
+# Hello World AWS Lambda and Amazon API Gateway REST API
+
+This project contains source code and supporting files for a serverless application that you can deploy with the AWS SAM CLI. It includes the following files and folders.
```

Comment:
I would add a note somewhere in the intro that the APIGW resource will be automatically created based on the Event mapping.